raster-foundry/raster-foundry
# Fix Log Levels for Backend

## Overview

Prior to this application a mix of log4j, slf4j, and logback were being used in the application due to dependencies. This PR explicitly removes log4j from our dependencies and uses slf4j and logback. As a result, all log4j.xml config files are deleted in this PR.

### Checklist


## Testing Instructions

- Re-assemble backsplash and api servers
- Start the server and interact with the UI, notice database logs are printed out
- Edit `docker-compose.yml` and set `RF_LOG_LEVEL` to `INFO`
- Restart the servers and interact with the app
- Notice less log output
